funnel/forms/project.py

```diff
@@ -172,15 +172,20 @@ def validate_bg_color(self, field):
 
 class CfpForm(forms.Form):
     instructions = forms.MarkdownField(
-        __("Call for proposals"),
+        __("Proposal guidelines"),
         validators=[forms.validators.DataRequired()],
         default='',
+        description=__(
+            "Set guidelines for the type of sessions (talks, workshops, other format) your project is accepting, your review process and any other info for participants"
```

Comment:
In Python this can be done like this:
```python
description=__(
    "This is a long line of text "
    "that continues into the next line. "
    "By convention the space is left at "
    "the end of the line and not the beginning "
    "of the next line."
    )
```

Since there are no commas, Python's parser will automatically concatenate these strings.